packages/storybook/src/nimble/select/select.stories.ts

```diff
@@ -91,6 +119,10 @@ To display placeholder text within the \`Select\` you must provide an option tha
 Any \`Select\` without a default selected option should provide placeholder text. Placeholder text should always follow the pattern "Select [thing(s)]", for example "Select country". Use sentence casing and don't include punctuation at the end of the prompt.
 `;
 
+const groupedDescription = `
+To group options in a \`Select\`, you can use the \`nimble-list-option-group\` element. This element should be placed within the \`Select\` and contain the \`nimble-list-option\` elements that you want to group. Note that a \`nimble-list-option-group\` placed within another \`nimble-list-option-group\` is not supported. The \`label\` attribute of the \`nimble-list-option-group\` element will be used as the group label. Alternatively, text can be provided next to the \`nimble-list-option-group\` element to serve as the group label.
```

Comment:
Can you use the exported tag names here rather than hard-code element tags in the description?